### ISSUE TITLE
Arguments for TEST_CODE

### ISSUE THREAD

**Issue by bunverdenz** [zuzkajelcicova/CS454-Automated_patching_using_GP#13]
![image](https://user-images.githubusercontent.com/33751306/48315529-dc0a4980-e61a-11e8-960e-7096cda6962b.png)
Here, GCD.java could have input from the shell, which I could give the value of a and b during the compilation, do you want me to do that? I mean, change the GCD.java to accept input from shell.
@zuzkajelcicova @Asdf11x @zele-git

**Comment by zuzkajelcicova** [comment]
@bunverdenz , no, that is not what we should do. I included this example code to test out whether the AST manipulation works. The GCD and also e.g. LeapYear, which are currently provided as resources have to be changed (just wrapped into methods with arguments). 
You should compile the code without giving it an input. You will firstly run the code and provide an input when you execute test cases (JUnit to compute fitness). There you have to have different test scenarios with different arguments that will be passed to the class.
